### PR TITLE
Add missing Paldean species data

### PR DESCRIPTION
**Pull request by ajhyndman**
Adds species data to gen 9 dex that was missed in the first pass.

NOTE: I re-enabled tests to find these missing pokemon and fill them in.  However, it's not obvious to me that we want to actually delete all of the previous generation pokemon from the SV dex.  It doesn't look like we removed old pokemon from the sword/shield dex, so I don't know why we would want to do so here.

I left the tests disabled for the moment, punting that decision down the road again.

### CODE REVIEW

File: calc/src/data/species.ts [hunk 1]
```diff
@@ -9126,14 +9154,36 @@ const SV_PATCH: {[name: string]: SpeciesData} = extend(true, {}, SS, {
...
-    abilities: {0: 'Thick Fat'},
+    abilities: {0: 'Lingering Aroma'},
+    otherFormes: ['Oinkologne-F'],
+  },
+  'Oinkologne-F': {
+    types: ['Normal'],
+    bs: {hp: 115, at: 90, df: 70, sa: 59, sd: 90, sp: 65},
+    weightkg: 120,
+    abilities: {0: 'Aroma Veil'},
+    baseForme: 'Oinkologne',
```
Comment (TheJetOU) [inline calc/src/data/species.ts:9165]:
```suggestion
    baseSpecies: 'Oinkologne',
```

File: calc/src/data/species.ts [hunk 2]
```diff
@@ -9126,14 +9154,36 @@ const SV_PATCH: {[name: string]: SpeciesData} = extend(true, {}, SS, {
...
+    weightkg: 60.2,
+    abilities: {0: 'Zero to Hero'},
+    otherFormes: ['Palafin-Hero'],
+  },
+  'Palafin-Hero': {
+    types: ['Water'],
+    bs: {hp: 100, at: 160, df: 97, sa: 106, sd: 87, sp: 100},
+    weightkg: 97.4,
+    abilities: {0: 'Zero to Hero'},
+    otherFormes: ['Palafin-Hero'],
```
Comment (TheJetOU) [inline calc/src/data/species.ts:9185]:
```suggestion
    baseSpecies: 'Palafin',
```

**Review by TheJetOU — COMMENTED** [review]
You also use `baseForme` instead of `baseSpecies` for the three Squawkabilly formes, three Paldean Tauors formes, and Paldean Wooper.

**Comment by ajhyndman** [comment]
@TheJetOU Whoops, great catches, thank you!

**Comment by wartab** [comment]
I do not currently have time to do a Pull Request for this, but if you could change a typo in Bellibolts typing. It says Electic instead of Electric. I'll create a Pull Request on Tuesday otherwise.

**Comment by ajhyndman** [comment]
@wartab nice find, I included Bellibolt's typing in an extra commit.

**Comment by wartab** [comment]
Another one: Farigiraf's second type is spelled Psyhcic instead of Psychic.

**Comment by wartab** [comment]
Also Annihilape says Fight instead of Fighting.

**Comment by AnnikaCodes** [comment]
Thanks!

**Pull request merged**